tox.ini

```diff
@@ -125,10 +126,30 @@ deps =
 commands =
     mkdir -p .report
     coverage run --source={[vars]src_path},testing/src/scenario \
-             -m pytest --ignore={[vars]tst_path}smoke -v --tb native {posargs}
+             -m pytest --ignore={[vars]tst_path}smoke --ignore={[vars]tst_path}test_benchmark.py \
+             -v --tb native \
+             -W 'ignore:Harness is deprecated:PendingDeprecationWarning' {posargs}
     coverage xml -o .report/coverage.xml
     coverage report
 
+[testenv:benchmark]
+description = Run benchmark tests
+passenv =
+    RUN_REAL_PEBBLE_TESTS
+    PEBBLE
+deps =
+    PyYAML==6.*
+    websocket-client==1.*
+    coverage[toml]~=7.0
+    pytest~=7.2
+    pytest-benchmark
+    typing_extensions~=4.2
+    jsonpatch~=1.33
```

Comment:
Do we need jsonpatch here?

Reply:
Right now, yes, because pytest goes through collecting all the tests and then pytest-benchmark marks lots of them as ones to skip - but we've still imported all the other test files where there are `import jsonpatch` lines.

If the test is in a separate folder instead, then I think that can be avoided.